styles.css

```diff
@@ -0,0 +1,551 @@
+  letter-spacing: 0.3700000047683716px;
+  text-align: left;
+  margin-left: 25px;
+}
+
+.text {
+  color: #f4f5f7;
+  height: 148px;
+  width: 327px;
+  font-family: Inter, serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-left: 25px;
+}
+
+.link {
+  align-self: flex-start;
+  margin-left: 20px;
+  height: 70px;
+  width: 165px;
+  font-family: Inter, serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  border: none;
+}
+
+.content {
+  margin-top: 0;
+  height: 100%;
+}
+
+.first,
+.second,
+.third {
+  display: flex;
+  flex-direction: column;
+  padding-top: 5px;
+}
+
+.first {
+  height: 290px;
+}
+
+.second,
+.third {
+  height: 340px;
+}
+
+.first h2 {
+  height: 50px;
+  width: 175px;
+  font-family: "InaiMathi", Roboto, sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.second h2,
+.third h2 {
+  font-family: Inter, serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.objects {
+  font-family: Inter, serif;
+  font-size: 22px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 43px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.rubric {
+  display: inline-flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-self: center;
+  width: 233px;
+  height: auto;
+}
+
+.objects .rubric {
+  margin: 0 auto;
+}
+
+.lang h2 {
+  margin-left: 15px;
+}
+
+.frame h2 {
+  margin-right: 15px;
+}
+
+.skill h2 {
+  margin-right: 60px;
+}
+
+.rubric img {
+  align-self: center;
+}
+
+.lang,
+.frame,
+.skill {
+  display: flex;
+  flex-direction: row;
+}
+
+.lang img {
+  height: 17px;
+  width: 17px;
+  left: 12px;
+  top: 20.7279052734375px;
+  border-radius: 0;
+}
+
+.frame img {
+  height: 18px;
+  width: 18px;
+  left: 12px;
+  top: 11px;
+  border-radius: 0;
+}
+
+.skill img {
+  height: 18px;
+  width: 30px;
+  left: 12px;
+  top: 11px;
+  border-right: 20px;
+  border-radius: 50px;
+}
+
+.bn-buttons .blue:hover {
+  background: lightblue;
+}
+
+.bn-buttons .orange:hover {
+  background: lightsalmon;
+}
+
+.bn-buttons .pink:hover {
+  background: lightpink;
+}
+
+.bn-buttons .red:hover {
+  background: lightcoral;
+}
+
+.css:hover {
+  background: #264de4;
+}
+
+.html:hover {
+  background: #9f4511;
+}
+
+.boot:hover {
+  background: #563d7c;
+}
+
+.ruby:hover {
+  background: #820c02;
+}
+
+.pr-button:hover {
+  background: #1817a2;
+}
+
+.resume:hover {
+  background: #0a6e2b;
+}
+
+.project:active {
+  background-color: #dc143c;
+}
+
+.link:active {
+  background-color: #f00;
+}
+
+.bn-list:active {
+  background-color: #9b111e;
+}
+
+.buttons:active {
+  background-color: #ff7f50;
+}
+
+.see-pr:active {
+  background-color: #b22222;
+}
```

Comment:
- [ ] [OPTIONAL] `see-pr` is a div, thus giving a background color on it is making issues: 

![chrome-capture](https://user-images.githubusercontent.com/77038610/144664993-4f30cc94-5928-4d98-b252-91bf12626ef1.gif)

As you can  see, there is some red thick border around the button, I recommend you to delete it. Instead you can try to target button not the container of the button 😉

Reply:
Done.